br/pkg/lightning/backend/backend.go

```diff
@@ -501,3 +512,7 @@ type EngineWriter interface {
 	IsSynced() bool
 	Close(ctx context.Context) (ChunkFlushStatus, error)
 }
+
+func (oe *OpenedEngine) GetEngineUuid() uuid.UUID {
+	return oe.uuid
+}
```

Comment:
```suggestion
func (engine *OpenedEngine) GetEngineUuid() uuid.UUID {
	return engine.uuid
}
```

It is better to keep the name of receiver consistent.